docs/personas/personas.qmd

```diff
@@ -2,9 +2,9 @@
 title: Personas
 ---
 
- - **Infrastructure Engineer**: I create Azure infrastructure via Terraform code, and I am also known as the platform administrator or maintainer.
+ - **Infrastructure Engineer/Administrator**: I create Azure infrastructure via Terraform code, and I am also known as the platform administrator or maintainer.
```

Comment:
Can see where you're coming from, but I think these personas are used throughout the documentation, so I'm not sure if having `/Administrator` makes this more complicated?